src/main/java/uk/gov/register/resources/VerifiableLogResource.java

```diff
@@ -29,6 +34,15 @@ public RegisterProof registerProof() {
 
     @GET
     @Path("/entry/{entry-number}/{total-entries}/merkle:sha-256")
+    public Response getProofRedirect(
```

Comment:
we need to do the redirect in this class as it owns the top level `/proof` paths.